### ISSUE TITLE
Aprender a trabalhar com a linguagem Markdown no GitHub

### ISSUE THREAD

**Issue by hyperiondeimos** [hyperiondeimos/PCCT2018#6]
Aprendam a mexer um pouquinho na sintaxe de operação do GitHub usando a Markdown.

**Comment by MViniciusSC** [comment]
É pra testar os labels?

**Comment by hyperiondeimos** [comment]
Não, apenas para se familiarizar com essa micro-linguagem de formatação para que você possa escrever comentários e até outras coisas legais aqui no GitHub.

**Comment by hyperiondeimos** [comment]
[Aprenda Markdown](https://blog.da2k.com.br/2015/02/08/aprenda-markdown/)
[Mastering Markdown](https://guides.github.com/features/mastering-markdown/)
[Como usar o Markdown](https://docs.microsoft.com/pt-br/contribute/help-crr/help-content/contribute/contribute-how-to-write-use-markdown)
